src/schema/v2/artwork/index.ts

```diff
@@ -1294,6 +1293,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       series: markdown(),
+      setVideoAsCover: {
```

Comment:
#minor should we be prefixing boolean fields with `is` ?

Reply:
It's less a check against whether a prop is true, but rather setter: 

<img width="739" alt="Screen Shot 2022-09-30 at 11 22 13 AM" src="https://user-images.githubusercontent.com/236943/193333061-722769b3-3297-4437-a747-dda1067f81e7.png">

I tend to understand `isFoo` is more of a one way street, a getter that refers to something like 

```tsx
const isThing = !!thing.length > 5
```

Reply:
OH i think i understand what you mean, since this is MP: we're not setting anything over here. I can update 👍